JEMS-CSE416/JEMS
# cypress selected page route test. cypress App component mounts test

`git fetch`
`git checkout cypress`

### Cypress Installation
1. cd to client
2. npm install cypress --save-dev

### Running Cypress
1. npx cypress open

### Testing
1. Select end to end testing
2. Select desired browser
3. Check routing tests pass
4. Go back and select component testing
5. Select desired browser
6. Check component test pass
